guides/common/modules/ref_best-practices-for-errata.adoc

```diff
@@ -0,0 +1,24 @@
+[id="best-practices-for-errata_{context}"]
+= Best practices for errata
+
+* {Project} shows which errata are applicable for each content host by comparing package lists with installed and available packages.
+* Use errata to add patches for security issues to a frozen set of content without unnecessarily updating other unaffected packages.
+* View errata on the content hosts page and choose between errata from the current content view and lifecycle environment and the _Library_ lifecycle environment which contains the latest synchronized packages.
++
+You can only apply errata included in the content view version of the lifecycle of your host.
+You can view this as a recommendation to promote content views.
+* You can subsequently add synchronized errata which are applicable to at least one content host to your content views.
```

Comment:
I am not sure it is a good idea to put a procedure in a reference module.

Reply:
Dropped. The procedure is already described in "managing errata".